Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -2,31 +2,35 @@ import Foundation
 
 @objcMembers
 public class SentryReplayOptions: NSObject, SentryRedactOptions {
-    
+
     /**
      * Enum to define the quality of the session replay.
      */
     @objc
     public enum SentryReplayQuality: Int {
+        static let names = [ "low", "medium", "high"]
```

Comment:
Instead of doing this where we have to maintain the indexing to the int values, let's add a conformance to `CustomStringConvertible` on the enum. That will be much more type safe, and logically safe as we would use a switch and those are compiler-checked for completeness. 

I see that's actually already added [here](https://github.com/getsentry/sentry-cocoa/pull/4495/files#diff-2659ecdf6eb3bd69676310e24f3344f77e2fa86765c77c6b6dd92ca22544f315R172), so I'd recommend just moving that to here, and using a switch instead of array indexing.

If for some reason you still need the array of all cases, you can also have it conform to `CaseIterable`.

Reply:
Thanks, I've copied `SentryLevel` which uses the array, I'll update it.